Source/Scene/Primitive.js

```diff
@@ -812,6 +820,11 @@ define([
                     this._state = PrimitiveState.COMBINING;
 
                     when(promise, function(packedResult) {
+                        if (!defined(packedResult)) {
+                            that._state = PrimitiveState.FAILED;
```

Comment:
What triggers this if block?  If zero geometry is created because all instances were invalid after deduplication?